.github/workflows/install_snippet.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -o errexit -o nounset -o pipefail
+
+# Set by GH actions, see
+# https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables
+TAG=${GITHUB_REF_NAME}
+
+
+cat << EOF
+## Install [Aspect CLI](https://www.aspect.build/cli)
+
+The underlying version of Bazel can be configured in your \`.bazelversion\` file or the \`BAZEL_VERSION\` environment variable.
```

Comment:
Ahh. This bit only applies when you use `.bazeliskrc` to bootstrap the Aspect CLI. If you homebrew install you can still use `.bazeliskrc` to configure the Bazel version.

Reply:
Ok I've moved that down to the bottom.